src/Model/Document/Editable/Outputchanneltable/MetaEntry/Table.php

```diff
@@ -54,7 +54,7 @@ public function setValues($values)
     {
         $this->values = $values;
 
-        $this->spanCleanedValues = [];
+        $this->spanCleanedValues = '';
```

Comment:
It's wrong there, likey has to stay `array`, please change the [property type](https://github.com/pimcore/web2print-tools/blob/8e383d2a72a1fa0c88a7f5cbfb0226c942099a70/src/Model/Document/Editable/Outputchanneltable/MetaEntry/Table.php#L28)

See also
https://github.com/pimcore/web2print-tools/blob/8e383d2a72a1fa0c88a7f5cbfb0226c942099a70/src/Model/Document/Editable/Outputchanneltable/MetaEntry/Table.php#L61
https://github.com/pimcore/web2print-tools/blob/8e383d2a72a1fa0c88a7f5cbfb0226c942099a70/src/Model/Document/Editable/Outputchanneltable/MetaEntry/Table.php#L74